### PR TITLE
DP-22983 Callout link component on org page not rendering correctly

### PR DESCRIPTION
**Pull request by Javi-er**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Description

Added a new "compact" variant to the Action Finder component to accommodate the organization section on OpenMass that doesn't use the component header and doesn't have a background color.
## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-22983)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: changelogs/DP-22983.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Changed:
+  - project: Patternlab
+    component: ActionFinder
+    description: Added a new action finder compact variant to accomodate action links with no header or background color.
```
Comment (clairesunstudio) [inline changelogs/DP-22983.yml:57]:
```suggestion
    description: Added a new action finder compact variant to accomodate action links with no header or background color. (#1512)
```

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/action-finder.twig [hunk 2]
```diff
@@ -5,30 +5,36 @@
...
     </style>
   {% endif %}
   <div class="ma__action-finder__container">
-    <header class="ma__action-finder__header">
-      <h2 class="ma__action-finder__title">
-        {{ actionFinder.title }}
-        {% if actionFinder.contextPositionTop and actionFinder.titleContext %}<span class="visually-hidden"> {{ actionFinder.titleContext }}</span>{% endif %}
-      </h2>
-    </header>
+    {% if not actionFinder.compact %}
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/action-finder.twig:30]:
```suggestion
    {% if not actionFinder.title %}
```

**Comment by clairesunstudio** [comment]
In order for this ActionFinder component to be more easily configured and reused, instead of adding a `compact` variant, can we use the existing `noBackground` check to conditionally render the background color and check for the `title` to conditionally render section title?

**Comment by Javi-er** [comment]
Closing and starting another one.

**Pull request closed without merging**